swagger/public_v1/api_spec.json

```diff
@@ -124,7 +124,7 @@
             "type": "string",
             "nullable": true,
             "format": "markdown",
-            "description": "Description of the accredited body for this course.",
+            "description": "Description of the accredited provider for this course.",
```

Comment:
There are some left overs here, from the `accredited_body` -> `accredited_provider` change